core/persistence-api/src/main/java/org/apache/syncope/core/persistence/api/dao/search/SearchCond.java

```diff
@@ -363,7 +363,7 @@ public boolean isValid() {
         boolean isValid = false;
 
         if (type == null) {
-            return isValid;
+            return false;
```

Comment:
What's the rationale behind such change?

Reply:
Same as above. Original variable name is always `false`

Reply:
See above.